src/Sylius/Bundle/SettingsBundle/Resources/config/doctrine/model/Settings.orm.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping">
+
+    <mapped-superclass name="Sylius\Bundle\SettingsBundle\Model\Settings" table="sylius_settings">
+        <unique-constraints>
+            <unique-constraint columns="schema_alias,namespace"/>
+        </unique-constraints>
+
+        <id name="id" column="id" type="integer">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="schemaAlias" column="schema_alias" type="string" nullable="false"/>
+        <field name="namespace" type="string" nullable="true"/>
+
+        <one-to-many target-entity="Sylius\Bundle\SettingsBundle\Model\ParameterInterface" mapped-by="settings"
+                     field="parameters" index-by="name" fetch="EAGER">
+            <cascade>
+                <cascade-persist/>
```

Comment:
Isn't the `CASCADE DELETE` that is in `Parameter.orm.xml` enough? Why would we also need to cascade a doctrine remove here?